src/components/views/dialogs/security/AccessSecretStorageDialog.tsx

```diff
@@ -403,6 +404,7 @@ export default class AccessSecretStorageDialog extends React.PureComponent<IProp
                             <input type="file"
                                 className="mx_AccessSecretStorageDialog_recoveryKeyEntry_fileInput"
                                 ref={this.fileUpload}
+                                onClick={chromeFileInputFix}
```

Comment:
Will this actually fire though? This element is not visible and is clicked programmatically, I know programmatic focus doesn't fire onFocus, not sure about onClick.

Reply:
From what I’ve been able to test, it works fine.

A sample case:

```html
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<script>
const testinput = document.querySelector("#testinput");
const testbutton = document.querySelector("#testbutton");
testinput.addEventListener("click", () => {
  console.log("Input has been clicked!");
});
testbutton.addEventListener("click", () => {
  testinput.click();
});
</script>
</head>
<body>
<input id="testinput" type="file">
<button id="testbutton">Test</button>
</body>
```

This works fine in Firefox and Chromium, so I’d expect it to work everywhere.